### PR TITLE
Check Progress Bar

### PR DESCRIPTION
**Pull request by RoyalSix**
Is dynamic and works as long as module not check status is "UNCHECKED" or "NOT_CHECKED".
Maybe we should create a standard for that in documentation.

### CODE REVIEW

File: src/js/components/core/ModuleProgress/ModuleProgressBar.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      },
+      label: "",
+      progress: 0,
+      max: 0
+    };
+  },
+
+  componentWillMount: function () {
+    CoreStore.addChangeListener(this.showProgress);
+    CheckStore.addEventListener('changedCheckStatus', this.updateProgress);
```
Comment (mannycolon) [inline src/js/components/core/ModuleProgress/ModuleProgressBar.js:27]: you might want to remove all the listeners in componentWillUnMount

**Comment by mannycolon** [comment]
Reviewed.
I don't have write access to the repo yet in order to merge this PR to develop, Im waiting on Chris. Don't forget to take a look at the comments I left. @RoyalSix

**Comment by klappy** [comment]
@mannycolon: You should now have write access.

**Comment by mannycolon** [comment]
i still dont have write access @klappy

**Comment by mannycolon** [comment]
I dont know why

File: src/js/components/core/ModuleProgress/ModuleProgressBar.js (hunk 1 above)
Comment (RoyalSix) [inline src/js/components/core/ModuleProgress/ModuleProgressBar.js:27]: added 👍

**Pull request merged**